<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowMap</class>
 <widget class="QWidget" name="ShowMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShowMap</string>
  </property>
  <widget class="QLabel" name="currentMapLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>A思源黑体—05</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前地图：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>658</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>A思源黑体—04</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image: url(:/new/prefix1/image/cancel.png)}
QPushButton:hover{border-image: url(:/new/prefix1/image/cancel.png)}
QPushButton:pressed{border-image: url(:/new/prefix1/image/cancel1.png)}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetBoardButton">
   <property name="geometry">
    <rect>
     <x>914</x>
     <y>344</y>
     <width>70</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image: url(:/new/prefix1/image/reset.png)}
QPushButton:hover{border-image: url(:/new/prefix1/image/reset.png)}
QPushButton:pressed{border-image: url(:/new/prefix1/image/reset1.png)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addBoardButton">
   <property name="geometry">
    <rect>
     <x>914</x>
     <y>166</y>
     <width>70</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image: url(:/new/prefix1/image/add.png)}
QPushButton:hover{border-image: url(:/new/prefix1/image/add.png)}
QPushButton:pressed{border-image: url(:/new/prefix1/image/add1.png)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>658</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>A思源黑体—04</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image: url(:/new/prefix1/image/ok.png)}
QPushButton:hover{border-image: url(:/new/prefix1/image/ok.png)}
QPushButton:pressed{border-image: url(:/new/prefix1/image/ok1.png)}</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lessBoardButton">
   <property name="geometry">
    <rect>
     <x>914</x>
     <y>255</y>
     <width>70</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image: url(:/new/prefix1/image/less.png)}
QPushButton:hover{border-image: url(:/new/prefix1/image/less.png)}
QPushButton:pressed{border-image: url(:/new/prefix1/image/less1.png)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mapIndexLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>A思源黑体—05</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>332</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="cruiseGroupBox">
   <property name="geometry">
    <rect>
     <x>485</x>
     <y>174</y>
     <width>296</width>
     <height>386</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#cruiseGroupBox
{
	border-image: url(:/new/prefix1/image/group4.png);
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="addItemButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>325</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>A思源黑体—04</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string>添加点</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="lessItemButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>325</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>A思源黑体—04</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string>删除点</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>80</y>
      <width>258</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>A思源黑体—05</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="cruiseGroupBox_returnBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>18</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/image//return.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cruiseLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>A思源黑体—05</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>巡航点</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>161</width>
     <height>304</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox
{
	border-image: url(:/new/prefix1/image/group.png);
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="autoButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>141</width>
      <height>134</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/prefix1/image/aout.png)}
QPushButton:hover{border-image: url(:/new/prefix1/image/aout.png)}
QPushButton:pressed{border-image: url(:/new/prefix1/image/aout1.png)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="manualButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>140</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/prefix1/image/manual.png)}
QPushButton:hover{border-image: url(:/new/prefix1/image/manual.png)}
QPushButton:pressed{border-image: url(:/new/prefix1/image/manual1.png)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
